src/core/lib/security/transport/secure_naming.c

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
I don't think that `secure_naming.c` is a good name for this file since it just deals with lb targets info. How about `lb_targets_info.c`?